### ISSUE TITLE
Enter key doesn't submit some forms

### ISSUE THREAD

**Issue by spantaleev** [zadam/trilium#553]
Reproducible on `0.32.2-beta`.

The "initial server setup" and "set up sync with server" forms can't be submitted by pressing the `Enter` key.

Doing various tests for debugging and setting up servers over and over causes me to notice this a bit too much. I guess regular users only need to log in once (usually) and don't see this too often.

Anyway, this is just a minor paper-cut type of issue that possible degrades the UX a bit for some people.

**Comment by zadam** [comment]
Yes, there is probably a lot of such issues, but as you mention these are usually things you need to setup once so I don't consider them as high priority.

**Comment by eliandoran** [comment]
Solved in TriliumNext in https://github.com/TriliumNext/Notes/pull/1071.
